libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/target/BriefcaseSyncDownTarget.java

```diff
@@ -174,6 +180,16 @@ public int cleanGhosts(SyncManager syncManager, String soupName, long syncId) th
         return countGhosts;
     }
 
+    @Override
```

Comment:
This is unrelated.
Without overriding this method a sync down with LEAVE_IF_CHANGED will not work correctly if the briefcase targets multiple object types.
(Need to add a test for that also).